stubs/tensorflow/tensorflow/core/example/example_parser_configuration_pb2.pyi

```diff
@@ -0,0 +1,127 @@
+"""
+@generated by mypy-protobuf.  Do not edit manually!
+isort:skip_file
```

Comment:
Huh, I hadn't noticed before that mypy-protobuf set these isort directives in the pyi files it generated. (No change requested!)